### ISSUE TITLE
ShowPassword icon inaccessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by cjkarle** [carbon-design-system/carbon#7381]
## Environment

> Operating system

Win 10 latest

> Browser

Edge 87.0, Firefox 83.0

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
The Show Password eyeball icon supplied by TextInput.PasswordInput is insufficiently visible to HCM users
<img width="427" alt="Screen Shot 2020-12-01 at 4 15 26 PM" src="https://user-images.githubusercontent.com/75339318/100800864-2d015c00-33f5-11eb-8a53-c8c572b30e0c.png">

> What version of the Carbon Design System are you using?

7.20.0
